theano/tensor/nnet/nnet.py

```diff
@@ -2335,3 +2335,30 @@ class of the input x while the second predicts the output of the input x in
         output_probs = target_class_probs * output_probs
 
     return output_probs
+
+
+def elu(x, alpha=1):
+    """
+    Compute the element-wise exponential linear activation function.
+
+    .. versionadded:: 0.8.0
+
+    Parameters
+    ----------
+    x : symbolic tensor
+        Tensor to compute the activation function for.
+    alpha : scalar
+
+
+    Returns
+    -------
+    symbolic tensor
+        Element-wise exponential linear activation function applied to `x`.
+
+    References
+    -----
+    .. [1] Djork-Arne Clevert,  Thomas Unterthiner, Sepp Hochreiter
+        "Fast and Accurate Deep Network Learning by
+        Exponential Linear Units (ELUs)" <http://arxiv.org/abs/1511.07289>`.
+    """
+    return alpha * (tensor.exp(0.5 * (x - abs(x))) - 1) + 0.5 * (x + abs(x))
```

Comment:
> Do you have an idea why they didn't included alpha? 

The formula in the research paper includes alpha (alpha is used to saturate negative net inputs). But, all the experiments mentioned in the paper used alpha = 1.  

> Is there a reason, why you didn't used the same algo as the version in the test in numpy?

I used `tensor.exp` because while reading the source code, I came across functions using `tensor.exp`. So, I didn't use numpy. 
But, I will change this formula to a simpler one which is in Lasagne, as you said.